source/MaterialXFormat/External/PugiXML/pugixml.cpp

```diff
@@ -3419,7 +3419,31 @@ PUGI__NS_BEGIN
 				{
 					mark = s; // Save this offset while searching for a terminator.
 
-					PUGI__SKIPWS(); // Eat whitespace if no genuine PCDATA here.
+					// Add a new node per new blank line, while eating whitespace
```

Comment:
Since we're making changes to the core PugiXML code, and will need to maintain them across future revisions, we should bracket these changes with MaterialX Begin/End comments, in the spirit of the single-line MaterialX comments found here:

https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/source/MaterialXFormat/External/PugiXML/pugixml.cpp#L3925